mkt/developers/templates/developers/includes/macros.html

```diff
@@ -125,21 +125,27 @@
   <a href="#" class="button toggle view-all">{{ _('View all') }}</a>
   <a href="#" class="button toggle hide-all hidden">{{ _('Hide all') }}</a>
   <ul class="feature-choices">
+    {% set has_configure_permission =  action_allowed('Apps', 'Configure') %}
     {% for field in form.required_api_fields() %}
       {% set key = field.name|replace('has_', '')|upper %}
       {% set tooltip = form.get_tooltip(field) %}
-      <li>
-        <label for="{{ field.auto_id }}">
-          {{ field }}
-          <span class="label">{{ field.help_text }}</span>
-          <div class="apis">
-            {% for api in mkt.constants.APP_FEATURES[key].apis %}
-              <code class="api">{{ api }}</code>
-            {% endfor %}
-          </div>
-          {% if tooltip %}<span class="hint">{{ tooltip }}</span>{% endif %}
-        </label>
-      </li>
+      {# Features without a tooltip are hidden from regular developers, since
```

Comment:
That might be better, yeah. Originally I didn't want to say it was "hidden" - it's there, it's just that we don't show it because it would not make sense to regular devs. But I think you're right and I should just explicitly say it's hidden.